README.md

```diff
@@ -110,36 +141,46 @@ If you are looking for the codebase of the current production version of GATK, p
   ./gatk-launch PrintReadsSpark -I input.bam -O output.bam
   ```
 
-#### Running GATK4 with inputs on Google Cloud Storage:
+#### <a name="jvmoptions">Passing JVM options to gatk-launch</a>
+
+* To pass JVM arguments to GATK, run `gatk-launch` with the `--javaOptions` argument: 
+
+    ```
+    ./gatk-launch --javaOptions "-Xmx4G" <rest of command>
+     
+    ./gatk-launch --javaOptions "-Xmx4G -XX:+PrintGCDetails" <rest of command>
+    ```
+
+#### <a name="gcs">Running GATK4 with inputs on Google Cloud Storage:</a>
 
-* GATK can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
+* Many GATK4 tools can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
   ```
   ./gatk-launch PrintReads -I gs://mybucket/path/to/my.bam -L 1:10000-20000 -O output.bam
   ```
 * You must set up your credentials first. There are three options:
```

Comment:
This line could use a bit more of a call-out -- maybe preface it with "Important note: " and bold it?